package.json

```diff
@@ -36,21 +36,21 @@
   ],
   "scripts": {
     "prepack": "nuxt-module-build",
-    "dev": "nuxt-module-build --stub && nuxi prepare playground && nuxi dev playground",
+    "dev": "nuxi dev playground",
```

Comment:
suggestion: I have specifically added the module build in the dev stop to avoid situation where you would run dev without first building your module. I don't think we should be removing this.. Maybe we could just call here `dev:prepare` instead.

Reply:
ok